ush/ioda/bufr2ioda/bufr2ioda_ncep_satwind_amv_goes.py

```diff
@@ -0,0 +1,530 @@
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+import sys
+
+sys.path.append('/work/noaa/da/eliu/JEDI-iodaconv/ioda-bundle/build/lib/')
+sys.path.append('/work/noaa/da/eliu/JEDI-iodaconv/ioda-bundle/build/lib/pyiodaconv/')
+sys.path.append('/work/noaa/da/eliu/JEDI-iodaconv/ioda-bundle/build/lib/python3.9/')
+
+import numpy as np
+import numpy.ma as ma
+from pyiodaconv import bufr
+import calendar
+import json
+import time
+import math
+import datetime
+from pyioda import ioda_obs_space as ioda_ospace
+
+# ====================================================================
+# Satellite Winds (AMV) BUFR dump file for GOES
+# ====================================================================
+# Subset    |  Spectral Band              |  Code (002023) |  ObsType
+# --------------------------------------------------------------------
+# NC005030  |    IRLW  (Freq < 5E+13)     |    Method 1    |   245
+# NC005031  |    WV Clear Sky/ Deep Layer |    Method 5    |   247
+# NC005032  |    VIS                      |    Method 2    |   251
+# NC005034  |    WV Cloud Top             |    Method 3    |   246
+# NC005039  |    IRSW  (Freq > 5E+13 )    |    Method 1    |   240
+# ====================================================================
+
+# Define and initialize  global variables
+global float32_fill_value
+global int32_fill_value
+global int64_fill_value
+
+float32_fill_value = np.float32(0)
+int32_fill_value = np.int32(0)
+int64_fill_value = np.int64(0)
+
+
+def Compute_WindComponents_from_WindDirection_and_WindSpeed(wdir, wspd):
```

Comment:
should we put this in a separate py script? I assume many of these converters will use the same function?

Reply:
we can tackle this in a future PR